website/react-magma-docs/src/pages/api/radio.mdx

```diff
@@ -470,20 +470,25 @@ export function Example() {
 Using React's `forwardRef` feature you can gain access to the reference of the internal radio.
 
 ```tsx
-import React from 'react';
+import React from "react";
```

Comment:
I just realized this example has been broken since 2.3.9, so it took me a second to understand what it's trying to achieve. I think it's trying to showcase how you can programmatically, with a ref (`radioRef`), set or unset the radio button option one. I think we can achieve this and clean this example a little bit better with something like this:

```
import React from 'react';
import {
  RadioGroup,
  Radio,
  Button,
  ButtonColor,
  ButtonGroup,
} from 'react-magma-dom';

export function Example() {
  const radioRef = React.useRef();
  const [radioGroupVal, setRadioGroupVal] = React.useState(null);

  function handleSelectOne() {
    if (!radioRef.current) {
      setRadioGroupVal(null);
    } else if (radioRef.current.checked) {
      setRadioGroupVal("");
    } else if (!radioRef.current.checked) {
      setRadioGroupVal("1");
    }
  }

  function handleFocusOne() {
    radioRef.current.focus();
  }

  return (
    <>
      <RadioGroup
        labelText="Forward ref usage"
        id="forwardRefGroup"
        name="forwardRef"
        value={radioGroupVal}
      >
        <Radio
          ref={radioRef}
          id="forwardRefRadio1"
          labelText="Option one label"
          value="1"
        />

        <Radio id="forwardRefRadio2" labelText="Option two label" value="2" />
      </RadioGroup>

      <ButtonGroup>
        <Button onClick={handleSelectOne}>Toggle option 1 selection</Button>
        <Button color={ButtonColor.secondary} onClick={handleFocusOne}>
          Focus option 1
        </Button>
      </ButtonGroup>
    </>
  );
}

```